smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/TypeScriptWriter.java

```diff
@@ -225,9 +225,9 @@ boolean writeShapeDocs(Shape shape, UnaryOperator<String> preprocessor) {
                     if (shape.getTrait(DeprecatedTrait.class).isPresent()) {
                         DeprecatedTrait deprecatedTrait = shape.expectTrait(DeprecatedTrait.class);
                         String deprecationMessage = deprecatedTrait.getMessage()
-                            .map(msg -> " " + msg)
-                            .orElse("");
-                        String deprecationString = "@deprecated" + deprecationMessage;
```

Comment:
```java
String deprecationMessage = deprecatedTrait.getMessage()
    .orElse("see description.");

String deprecationAnnotation = "@deprecated " + deprecationMessage;

docs = docs + "\n\n" + deprecationAnnotation;
```

Reply:
goal is to always have a message after the `@deprecated` annotation so that any text coming after is not mistakenly interpreted to be the deprecation message.

the tsdoc spec requires that `@deprecated` annotation comes with a message.